sound/smooth_swing_v2.h

```diff
@@ -95,6 +104,11 @@ class SmoothSwingV2 : public SaberBasePassThrough {
     // degrees per second
     // May not need to smooth gyro since volume is smoothed.
     float speed = sqrtf(gyro.z * gyro.z + gyro.y * gyro.y);
+    if (millis() - last_swing_millis_ >= 10 ) {
```

Comment:
doing this every 10 ms is pretty coarse, I like the other acceleration calculation better.

Reply:
Agreed, this is old code and needs to be replaced.

Reply:
removed, replaced with code in hybrid_font.h